client/src/app/ui/modules/editor/components/editor/editor.component.html

```diff
@@ -1 +1,264 @@
-<editor *ngIf="hasInitialized | async" [init]="getEditorSettings()" [formControl]="contentForm"></editor>
+<div class="editor">
+    <div *ngIf="editorReady" class="editor-menu">
+        <div class="line">
+            <div class="button-group">
+                <button
+                    mat-ripple
+                    class="text"
+                    [ngClass]="{ active: editor.isActive('heading') }"
+                    (click)="editor.chain().focus().toggleHeading({ level: selectedHeadingLevel }).run()"
+                    translate
+                >
+                    Heading {{ selectedHeadingLevel }}
+                </button>
+
+                <button mat-ripple [matMenuTriggerFor]="headingMenu">
+                    <mat-icon>arrow_drop_down</mat-icon>
+                </button>
+            </div>
+
+            <mat-menu #headingMenu="matMenu">
+                <button
+                    mat-menu-item
+                    [ngClass]="{ active: editor.isActive('heading', { level: 1 }) }"
+                    (click)="editor.chain().focus().toggleHeading({ level: 1 }).run()"
+                    translate
+                >
+                    Heading 1
+                </button>
+
+                <button
+                    mat-menu-item
+                    [ngClass]="{ active: editor.isActive('heading', { level: 2 }) }"
+                    (click)="editor.chain().focus().toggleHeading({ level: 2 }).run()"
+                    translate
+                >
+                    Heading 2
+                </button>
```

Comment:
Instead of writing basically the same element 6 times, using ngFor may make it more readable.

Same issue with the other button functionalities in this class.

Also: Things like `editor.chain().focus().toggleHeading({ level: selectedHeadingLevel }).run()` should be turned into class methods with descriptive names.